mod/gc_mobile_api/models/blog.php

```diff
@@ -69,6 +85,39 @@
  false
 );
 
+function foreach_blogs($blogs, $user_entity, $lang)
```

Comment:
If this will be re-used in other models, can we move to the generic `inc/functions.php` file? Thanks.